rmgpy/kinetics/arrhenius.pyx

```diff
@@ -249,7 +249,23 @@ cdef class Arrhenius(KineticsModel):
 
         # Set the rate parameter to a cantera Arrhenius object
         ctReaction.rate = self.toCanteraKinetics()
-    
+
+    cpdef ArrheniusEP toArrheniusEP(self):
```

Comment:
In order to convert Arrhenius to ArrheniusEP with a nonzero alpha you would need to know the ∆H of the reaction, which presumably is not known when this is called.  I think it would probably be not only an unused feature, but an almost-impossible-to-use feature.